index.html

```diff
@@ -357,91 +369,7 @@ <h1 id="logoImage"><a href="https://www.r2accelerator.org/"><img src="static/ima
-                                        <i class="fa fa-filter"></i>
-                                        <i class="fa fa-times"></i>
-                                        <span></span>
-                                    </div>
-                                    <div class="col sqs-col-12 choices-box choices-step-1 hidden" data-step="1">
-                                    </div>
-                                </div>
-
-                                <div class="col sqs-col-12 span-12 filter-box" data-step="2">
-                                    <div class="col sqs-col-12 filter-title-box">
-                                        <span class="filter-title">
-                                            <i class="fa fa-angle-right m-r-sm angle"></i>
-                                            Technology
-                                            <i class="fa fa-cogs m-l-sm"></i>
-                                        </span>
-                                    </div>
-                                    <div class="col sqs-col-12 selected-filter selected-filter-2 hidden" data-step="2">
-                                        <i class="fa fa-filter"></i>
-                                        <i class="fa fa-times"></i>
-                                        <span></span>
-                                    </div>
-                                    <div class="col sqs-col-12 choices-box choices-step-2 hidden">
-                                    </div>
-                                </div>
-
-                                <div class="col sqs-col-12 span-12 filter-box" data-step="3">
-                                    <div class="col sqs-col-12 filter-title-box">
-                                        <span class="filter-title">
-                                            <i class="fa fa-angle-right m-r-sm angle"></i>
-                                            Location availability
-                                            <i class="fa fa-map m-l-sm"></i>
-                                        </span>
-                                    </div>
-                                    <div class="col sqs-col-12 selected-filter selected-filter-3 hidden" data-step="3">
-                                        <i class="fa fa-filter"></i>
-                                        <i class="fa fa-times"></i>
-                                        <span></span>
-                                    </div>
-                                    <div class="col sqs-col-12 choices-box choices-step-3 hidden">
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-
-                        <div class="col sqs-col-8 vendors-list">
-                        </div>
-                    </div>
-                </div>
+                <!-- Content Dynaically Loaded -->
```

Comment:
Typo in the comment dynamically

Reply:
Hey, thanks for pointing this out. I have just pushed an update that should address this typo. 

In terms of the menu placement, I wasn't sure where the pages should be linked so I had reached out to ask and this location was the response. While I think the vendor database would be a great place to find a solution to reach out and ask us about, I can also see counter arguments where it could be a resource as well. I am open to updating the location. But, if we do, we should loop Miriam and Arend so everyone is on the same page.

Reply:
Both under Resources looks good to me, but you added Vendor Database under "Work With Us" that one should be wrong right? Or I am mistaken?

![screenshot from 2018-08-06 07-49-33](https://user-images.githubusercontent.com/492577/43699049-4b3f329a-994d-11e8-99fe-435e2e4e0f1b.png)

Reply:
Sure thing. Let me loop in Arend and Miriam as it was in the email chain with them that I was told to put that link under the "Work With Us" section. I will add you to the chain so we can discuss and all be on the same page. 

Would you mind sharing your email address with me so that I am able to add you? If everyone is in agreement, I will promptly update and share changes for your review.

Reply:
Ok never mind, I didn't know it was directly requested like that. Then If Arend and Miriam are ok with it +1 to deploy it. I thought it was a copy/paste mistake, my bad.